config/config.yaml

```diff
@@ -1,4 +1,22 @@
-dune_query_id: 2446884
-polling_rate_minutes: 2 # Polling interval in minutes
+dune_query_id: 3993272  # The Dune query ID used for fetching data
+second_dune_query_id: 3993271
+gather_to_start_dune_query_id: 3992306
```

Comment:
Unclear names. I think you only need two queries (and should name the parameters more expressively)
- All MEV Blocker bundle per block
- Confirmed transactions per block